### PR TITLE
Release SonarQube 10.5.1

### PR DESCRIPTION
**Pull request by davividal**

### CODE REVIEW

**Comment by davividal** [comment]
WIP as we are waiting for https://github.com/docker-library/official-images/pull/16690 to be merged.

File: google-cloud-marketplace-k8s-app/schema.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ x-google-marketplace:
   publishedVersion: "$TAG"
   publishedVersionMetadata:
     releaseNote: >-
-      Releasing SonarQube 10.4.x as a google cloud k8s marketplace application
+      Releasing SonarQube 10.5charts/sonarqube/values.yaml.x as a google cloud k8s marketplace application
```
Comment (carminevassallo) [inline google-cloud-marketplace-k8s-app/schema.yaml:10]: Typo here

**Review by carminevassallo — CHANGES_REQUESTED** [review]
Hey Davi,

Thanks for opening the PRs. I noticed a few improvements to make:

- In both charts, we should add the change about "Upgrade SonarQube to 10.5.1". So far, we only have the chart's update.
- a small typo on gcp

File: google-cloud-marketplace-k8s-app/schema.yaml (hunk 1 above)
Comment (davividal) [reply google-cloud-marketplace-k8s-app/schema.yaml:10]: Ops! `Ctrl+V` in the wrong window...

**Review by carminevassallo — APPROVED** [review]
LGTM 🚀

**Pull request merged**